.changeset/lovely-bulldogs-jam.md

```diff
@@ -0,0 +1,8 @@
+---
+'@leafygreen-ui/date-picker': minor
+---
+
+[LG-2930](https://jira.mongodb.org/browse/LG-2930)
+
+- Added stories for error state
+- `FormField` styling changes apply to `DatePicker`
```

Comment:
Can you add a little detail of what style changes were made?

Reply:
linked to the `FormField` changelog hyperlink that will be added in the future for both date picker and text input